papers/core_bench.md

```diff
@@ -0,0 +1,98 @@
+
+They reproduced some published works to generate the ground truth of the different tasks. To give some numerical room, they sampled by repeating each manual run three times, which was the ground truth for the 95 % prediction interval.
+
+![Figure taken from by Siegel et al. paper [@siegel2024corebenchfosteringcredibilitypublished]. In the Figure *(a)* it is detailed an example for a medium-level task. The figure *(b)* illustrates how the evaluation is performed.](core_bench_papers/core_bench_3.png)
+
+Similarly to other benchmarks such as MLAgentBench [@huang2024mlagentbenchevaluatinglanguageagents] or MLE-BENCH [@chan2024mlebenchevaluatingmachinelearning], they provide the agent with access to different files and the agent need to read and run those files to reach the final answer.
+
+Interestingly, they evaluate two different agents: AutoGPT and CORE-Agent. The first one is a general agent widely used, while the second is an adapted, improved version of the first one.
+
+## Results
+
+The general results of the evaluation are shown in the following table:
+
+| Agent      | LLM         | CORE-Bench-Medium | CORE-Bench-Hard |
+| :---:      | :--:        | :---------------: | :-------------: |
+| CORE-Agent | GPT-4o      | **0.57**          | **0.21**        |
+|            | GPT-4o-mini | 0.32              | 0.16            |
+|            |             |                   |                 |
+| AutoGPT    | GPT-4o      | 0.37              | 0.06            |
+|            | GPT-4o-mini | 0.02              | 0.02            |
+
+: Accuracy in the most difficult partitions of the benchmark {#tbl-letters}
+
+The number of models that they tested is limited. However, some important conclusions can be taken from there.
+
+As expected and the most obvious is that the performance decreases as the difficulty of the task increases. Also, the bigger model shows better results than the smaller ones, and the adapted agent shows better results than the general one, specially for the smallest or weakest model.
+
+They propose a case study in which they study the relation of the agents' performance with the economical cost of the models. However, this has not much sense since the models powering the agents are not the same, and they have very different costs per tokens expended.
+
+The most interesting result these days is that the models perform much better in the text than in the images. They claim that this is the case because the vision tasks are more complex, and the agents are not yet able to perform them. However, in my opinion, more ablations would be needed to clarify this point.
+
+### Additional results
+
+- Python tasks' results are better than R tasks' results.
+- Agents struggle to retrieve results from many files.
+- Better safety measures and guardrails are needed to prevent agents from running malicious code when commercially deploying these agents.
```

Comment:
```suggestion
- The results of the Python tasks are better than those of the R tasks.
- Agents struggle to retrieve results from many files.
- Better safety measures and guardrails are needed to prevent agents from running malicious code when commercially deploying these agents.
```